aiokafka/record/_crc32c.py

```diff
@@ -97,7 +98,9 @@
 _MASK = 0xFFFFFFFF
 
 
-def crc_update(crc, data):
+def crc_update(
+    crc: int, data: Union["array.array[int]", bytes, bytearray, Iterable[int]]
```

Comment:
Why so strange union? Shouldn't it be just `Buffer`?

Reply:
array.array() requires bytes | bytearray, see https://docs.python.org/3/library/array.html#array.array

Reply:
`Iterable[int]` covers all these types
```suggestion
    crc: int, data: Iterable[int]
```